src/FixedDataTable.js

```diff
@@ -364,14 +366,27 @@ var FixedDataTable = React.createClass({
     );
     this._touchHandler = new ReactTouchHandler(
       this._onScroll,
-      touchEnabled && this._shouldHandleWheelX,
-      touchEnabled && this._shouldHandleWheelY,
+      this._shouldHandleTouchX,
+      this._shouldHandleTouchY
       props.stopScrollPropagation
     );
 
     this.setState(this._calculateState(props));
   },
 
+  componentWillUnmount() {
+    this._wheelHandler = null;
```

Comment:
Is there any risk of the onWheel still firing when we unmount?  Or should we be okay since the DOM object is expected to be gone?